src/Configuration/src/ConfigServer/ConfigServerDiscoveryService.cs

```diff
@@ -99,11 +102,11 @@ internal IEnumerable<IServiceInstance> GetConfigServerInstances()
         return instances;
     }
 
-    internal async Task ProvideRuntimeReplacementsAsync(IDiscoveryClient discoveryClientFromDI, ILoggerFactory loggerFactory)
+    internal async Task ProvideRuntimeReplacementsAsync(IDiscoveryClient discoveryClientFromDI)
```

Comment:
I'm not sure what you mean. I just removed an unused parameter.

Reply:
Sorry, I was kind of musing there... This situation is either a bug, introduced in #803 that might take some effort to sort out (because I don't recall exactly how the BootstrapLoggerFactory works) or more code (like this param) should have been removed in that PR... https://github.com/SteeltoeOSS/Steeltoe/pull/803/files#diff-48f80b415dfa3346eb6f5d0b9c5c61ff4a888b05bdc1518ab8f269cf7228a936L116

Reply:
This pretty clearly goes beyond the scope of this PR, but we should revisit in #1015 before shipping 4.0